code/controllers/master.dm

```diff
@@ -72,6 +74,14 @@ var/global/datum/controller/master/Master = new()
 		return
 	world << "<span class='boldannounce'>Initializing subsystems...</span>"
 
+	//Seed ruins in space
+	/*
+	var/space_to_seed = SPACE_LEVELS_TO_SEED //Maybe this should be a config option?
+	while(space_to_seed)
+		var/target_z = pick(ZLEVEL_SPACEMAX, ZLEVEL_ABANDONNEDTSAT, ZLEVEL_STATION, ZLEVEL_DEEPSPACE)
+		seedRuins(RUINS_PER_LEVEL, target_z) //As well as this
+		space_to_seed--
+	*/
```

Comment:
Please don't just throw this into the MC. Make it a new subsystem or at the very least offload it to a proc or something, the MC is cluttered enough as is.